examples/04-cxx14/main.cpp

```diff
@@ -0,0 +1,31 @@
+/* 
+ * Source C++ test from: https://stackoverflow.com/a/30940639/51789 
+ */ 
+#include <iostream>
```

Comment:
There is no need to create separate example, you can just add this code to old example with condition `#if __cplusplus >= 201402L`.